tests/bearlib/languages/documentation/DocumentationExtractionTest.py

```diff
@@ -84,70 +96,90 @@ def test_extract_documentation_CPP(self):
+            (DocumentationComment(
+                ("\n"
+                 " This is the main function.\n"
+                 " @returns Exit code.\n"
+                 "          Or any other number.\n"),
+                language, docstyle,
+                docstyle_CPP_doxygen.markers[0],
+                TextRange.from_values(4, 1, 8, 4)),
+             DocumentationComment(
+                (" foobar\n"
+                 " @param xyz\n"),
+                language, docstyle,
+                docstyle_CPP_doxygen.markers[0],
+                TextRange.from_values(15, 1, 17, 4)),
+             DocumentationComment(
+                " Some alternate style of documentation\n",
+                language, docstyle,
+                docstyle_CPP_doxygen.markers[4],
+                TextRange.from_values(22, 1, 23, 1)),
+             DocumentationComment(
+                " ends instantly",
+                language, docstyle,
+                docstyle_CPP_doxygen.markers[0],
+                TextRange.from_values(26, 5, 26, 26)),
+             DocumentationComment(
+                (" Should work\n"
+                 "\n"
+                 " even without a function standing below.\n"
+                 "\n"
+                 " @param foo WHAT PARAM PLEASE!?\n"),
+                language, docstyle,
+                docstyle_CPP_doxygen.markers[4],
+                TextRange.from_values(32, 1, 37, 1))))
 
     def test_extract_documentation_CPP_2(self):
         data = DocumentationExtractionTest.load_testdata("data2.cpp")
 
-        docstyle_CPP_doxygen = DocstyleDefinition.load("CPP", "doxygen")
+        language = "cpp"
+        docstyle = "doxygen"
+
+        docstyle_CPP_doxygen = DocstyleDefinition.load(language, docstyle)
 
-        self.assertEqual(tuple(extract_documentation(data, "CPP", "doxygen")),
-                         (DocumentationComment(
-                             ("module comment\n"
-                              " hello world\n"),
-                             docstyle_CPP_doxygen.markers[0],
-                             TextRange.from_values(1, 1, 3, 4)),))
+        self.assertEqual(tuple(
+            extract_documentation(data, language, docstyle,)),
+            (DocumentationComment(
+                ("module comment\n"
+                 " hello world\n"),
+                language, docstyle,
+                docstyle_CPP_doxygen.markers[0],
+                TextRange.from_values(1, 1, 3, 4)),))
 
     def test_extract_documentation_PYTHON3(self):
         data = DocumentationExtractionTest.load_testdata("data.py")
 
-        docstyle_PYTHON3_default = DocstyleDefinition.load("PYTHON3",
-                                                           "default")
-        docstyle_PYTHON3_doxygen = DocstyleDefinition.load("PYTHON3",
-                                                           "doxygen")
+        language = "python3"
+        docstyle_default = "default"
+        docstyle_doxygen = "doxygen"
```

Comment:
hmm would you just inline them?
1. the strings itself are shorter
2. people don't need to look up the variables when reading the test :+1: